configuration/pih/petl/jobs/hiv/viral_load/source.sql

```diff
@@ -116,8 +126,9 @@ SELECT
         vl_result_date,
         specimen_number,
         vl_test_outcome,
-        IF(vl_result_detectable IS NOT NULL, 1, NULL) vl_result_detectable,
+        IF(vl_test_outcome_coded = @detected_viral_load, 1, 0) vl_result_detectable,
```

Comment:
So I added vl_result_detectable back in, but it's now based on the coded vl_test_outcome.